qid-batch-runner.yml

```diff
@@ -63,12 +63,44 @@ spec:
               secretKeyRef:
                 name: db-credentials
                 key: password
+          - name: SFTP_DIRECTORY
+            value: "upload/"
```

Comment:
we are sharing a single SFTP server between CI, White Lodge, Black Lodge and our own GCP projects, so there will be clashes if these tests run at the same time. We now have a config map `called project-config` which has the key-value pair `project-name=<project name>`. We should use this project name in the SFTP path, so that the different projects don't clash with each other, causing test failures.

Reply:
@NickGrantONS The acceptance tests shouldn't be concerned with _where_ the files are stored as long as they _are_ stored somewhere and can be found (and subsequently removed).

But, agreed, for actual usage of the remote SFTP each GCP project should have its own directory. I half-remember perhaps @kierenbeckett mentioning that each user would be logged in and presented with its own home directory? Or is this not the case?

Reply:
Each user will have its own home directory, but I presume you only have a test and prod user and are currently sharing the test one between whitelodge/blacklodge? @jedimasterjonny thoughts?

Reply:
Each user has it's own home directory so rm_test and rm_prod have distinct folders. 

However as there's only one SFTP stack, this spans all environments - so files dropped into rm_test on whitelodge, will be appear in the same folder on blackloge, CI, personal projects etc.

Reply:
yes, the problem is that the `rm_test` account is shared between CI, Black Lodge and White Lodge (any any personal projects which are whitelisted) so we have to use the project name prefix

Reply:
Thanks, all!